createInvoice/src/main/java/com/cloudelements/demo/usecase/eventhandling/APIResponseController.java

```diff
@@ -0,0 +1,51 @@
+package com.cloudelements.demo.usecase.eventhandling;
+
+import java.io.IOException;
+
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.cloudelements.demo.model.APIEvent;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/*
+ * This restcontroller allows the app to always listen for incoming EVENTS.
+ * Meaning upon element authentication you can just plug in your_url:8080/apiResponse
+ * 
+ * As our event structure is standard across the platform, the handleAPIResponse method will parse
+ * the retrieved json into an APIEvent object
+ */
+
+@RestController
+public class APIResponseController {
+
+	@RequestMapping(value="/apiResponse", method=RequestMethod.POST)
+	public void handleAPIResponse(@RequestBody String payload) throws ParseException, JsonParseException, JsonMappingException, IOException {
+		System.out.println("*** Received event *** \n" + payload );
```

Comment:
Spring Boot supports logging so it would be nice to use that rather than System.out.println

Reply:
Added springboot logger to the pom and changed all sysouts to logger.debug